Virgo-submit-infrastructure.yaml

```
@@ -0,0 +1,27 @@
+  
+Production: false
+SupportCenter: Self Supported
+
+GroupDescription: This is a cluster of submission hosts for the IGWN. 
+GroupID: <ID>
+
+Resources:
+  Virgo-submit-infrastructure:
+    Active: false
+    Description: stro.nikhef.nl is a glidein frontend node. 
+    ID: <ID>
```

Please us resource id:
```suggestion
ID: 1027
```